src/plrmodes/reimu_a.c

```diff
@@ -699,7 +699,7 @@ static double reimu_spirit_property(Player *plr, PlrProperty prop) {
 
 PlayerMode plrmode_reimu_a = {
 	.name = "Yōkai Buster",
-	.description = "Sometimes it is easier to let your resolve do the aiming.",
+	.description = "The tried-and-true: homing bullets and sharp needles. You don’t have the luxury of time to think about things.",
```

Comment:
Maybe something like this?
```suggestion
	.description = "The tried-and-true: homing amulets and sharp needles. You don’t have the luxury of time to think about the aiming.",
```